Corona-Warn-App/src/main/java/de/rki/coronawarnapp/eventregistration/checkins/qrcode/VerifiedTraceLocation.kt

```diff
@@ -0,0 +1,18 @@
+package de.rki.coronawarnapp.eventregistration.checkins.qrcode
+
+import android.os.Parcelable
+import de.rki.coronawarnapp.server.protocols.internal.pt.TraceLocationOuterClass
+import kotlinx.parcelize.Parcelize
+import org.joda.time.Instant
+
+@Parcelize
+data class VerifiedTraceLocation(
```

Comment:
We have `DefaultTraceLocation` that you can use here instead of creating a new model. It has the same properties.

Reply:
We also need the `signature`, because we need to store it in the db after verifying the qr code at event creation / checkin